R/govList.R

```diff
@@ -0,0 +1,82 @@
+#' @rdname layouts
```

Comment:
please could we also change the .R file name to gov_list so it matches the function name? 😄